server/src/index.js

```
@@ -30,6 +30,10 @@ const resolvers = {
 const typeDefs = importSchema(path.resolve('src/schema.graphql'))
 
 const server = new ApolloServer({
+  formatError: error => {
+    console.log(error);
+    return new Error('Internal server error');
```

Does this override the default error behaviour? In that case, won't it hide the actual source of the error from the front-end?

---

You're right, it does. I'll test out better options tonight. Thanks @brortao!

---

I still can't test, but I think this should be more helpful. It passes the error and response to the console. I'd expect this to bubble up pgsql and Prisma errors, but need to make sure.

```
formatError: error => {
    console.log(error);
    return error;
  },
formatResponse: response => {
  process.stdout.write(response);
  return response;
  },
```